nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -468,27 +472,68 @@ define([
          * Update the internal readiness state for the app cell parameters. When all fileTypes have
          * a "complete" state, then they are all completely filled out and ready to run.
          * @param {string} fileType - which filetype's app state to update
-         * @param {string} state - what the new ready state should be - one of complete, incomplete, error
+         * @param {string} newState - what the new ready state should be - one of complete, incomplete, error
          */
-        function updateParameterState(fileType, state) {
-            model.setItem(['state', 'param', fileType], state);
+        function updateParameterState(fileType, newState) {
+            model.setItem(['state', 'param', fileType], newState);
             const newFileTypeState = {};
-            for (const [fileId, state] of Object.entries(model.getItem(['state', 'param']))) {
-                newFileTypeState[fileId] = state === 'complete';
+            for (const [fileId, fileState] of Object.entries(model.getItem(['state', 'param']))) {
+                newFileTypeState[fileId] = fileState === 'complete';
             }
-            fileTypePanel.updateState({ completed: newFileTypeState });
+
+            state.fileType.completed = newFileTypeState;
             let cellReady = true;
             for (const state of Object.values(model.getItem('state.param'))) {
                 if (state !== 'complete') {
                     cellReady = false;
                     break;
                 }
             }
+            const uiState = cellReady ? 'editingComplete' : 'editingIncomplete';
+            updateState(uiState);
             if (cellReady) {
-                console.log('cell ready!');
+                buildPythonCode();
+            } else {
+                clearPythonCode();
             }
         }
 
+        function buildPythonCode() {
```

Comment:
follows the same pattern as other app cells. Get the cell id, make a run id, and send parameters to get Pythonified.